src/drone_controller/input_layer/drone_state_mapper.py

```diff
@@ -0,0 +1,18 @@
+"""
+Mapper Module
+"""
+
+
+# pylint: disable=too-few-public-methods
+# keep this only until this is implemented
+class DroneStateMapper:
+    """
+    Maps different inputs to an expected drone state.
+    """
+
+    def keyboard(self, user_input):
+        """
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Multi-line docstring summary should start at the first line](https://app.codacy.com/gh/treeandsea/DroneController/pullRequest?prid=5205179)